### DISCUSSION TITLE
Import/export materials

### DISCUSSION THREAD

**Discussion by MiguelSC98** [Anura3D/Anura3D_OpenSource#120] [Q&A] [answered]
Dear all.
Hi.
I have a question. Is it possible to import/export soil properties from one to another model? If it's correct. Please, can someone detail the steps to do it?

Thank you in advance for your help.

**Comment by murphy-john** [comment, marked as answer]
Will you explain more about what you mean by import and exporting the properties? Which properties? Which format would the properties be imported or exported? When would you like to do the import/export?
